admin/src/Components/CollectionTable.js

```diff
@@ -179,7 +179,7 @@ const CollectionTable = (props) => {
                     <div className="loadingOverlay">
                         <div className="loadingWrapper">
                             <span>
-                                <div uk-spinner="" />
+                                <div data-uk-spinner="" />
```

Comment:
Does this really work? I was under the impression that UiKit used non-standard attributes without the `data-` prefix.

Reply:
According to the docs it should:
![image](https://github.com/user-attachments/assets/051c29ad-6b5f-431f-a97b-25bb3f6e4c37)

Reply:
Great!